Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04
+
+RUN apt-get update
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get install -y python3 python3-pip apt-utils libgl1-mesa-glx libglib2.0-0
+
+WORKDIR /wf-process-pose-data
+
+COPY . .
+
+RUN yes | pip3 install -r requirements.txt
```

Comment:
`RUN pip install -e .`

Reply:
this will install all the dependencies and make this package available to python.